py-pkg/dummy_data/load_dummy_data.sh

```diff
@@ -0,0 +1,31 @@
+#! /bin/sh
```

Comment:
in the issue about CI, can you explicitly call out all the files currently under `py-pkg` which are redundant with what's currently available for CI in this repo? Basically so whoever picks up the issue knows what stuff to go clean up.

Reply:
updated #85